doc/reference_measure.xml

```diff
@@ -4257,6 +4257,57 @@ FROM ST_GeogFromText('MULTIPOLYGON(((-71.1044543107478 42.340674480411,-71.10445
 	  </refsection>
 	</refentry>
 
+	<refentry id="ST_Point_Inside_Circle">
```

Comment:
can we globally change ID to match the non-deprecated signature? it's becoming a part of URL when published, pointing people to wrong version.

Reply:
Good idea, will change this.